files/default/neutron-ha-tool.py

```diff
@@ -1061,48 +1061,53 @@ def pick(self):
         )
 
 
-class RemoteRouterNsCleanup(object):
-
-    def __init__(self, host, routerid):
+class RemoteNodeCleanup(object):
+    def __init__(self, host, timeout=10):
         self.target_host = host
-        self.namespace = "qrouter-" + routerid
-        self.timeout = 10
-        self.netns_del = "ip netns delete " + self.namespace
-        self.netns_pids = "ip netns pids " + self.namespace
+        self.timeout = timeout
+        self.netns_del = "ip netns delete "
+        self.netns_pids = "ip netns pids "
         self.netns_list = "ip netns list"
 
+    def _ssh_connect(self):
+        self.ssh_client = paramiko.SSHClient()
+        self.ssh_client.set_missing_host_key_policy(
+            paramiko.AutoAddPolicy())
+        self.ssh_client.load_system_host_keys()
+        self.ssh_client.connect(self.target_host, timeout=self.timeout)
+
     def _simple_ssh_command(self, command):
         # Note, that when get_pty is True, paramiko will never return anything
         # on the stderr channel, that's why we ignore it here. (stderr output
         # will endup in the stdout channel)
         _, stdout, _ = self.ssh_client.exec_command(command,
-                                                    timeout=10,
+                                                    timeout=self.timeout,
                                                     get_pty=True)
         out_lines = stdout.readlines()
         rc = stdout.channel.recv_exit_status()
         return [rc, [line.strip() for line in out_lines]]
 
-    def _namespace_exists(self):
+    def _namespace_exists(self, namespace):
```

Comment:
If that's the only reason then I'd say it's a false economy, since technical debt always increases in cost over time.  And @matelakat is currently blazing the trail and showing us all that it's perfectly possible to write really nice code under time constraints, which is quite inspiring :-)  But ultimately it's your call.  I don't think I want to give a -1 or a +1 right now.  Everything else here looks good to me though.